Before-Using-Agoric.md

```diff
@@ -0,0 +1,76 @@
+
+# Before Using Agoric Software
+
+Before working with the Agoric CLI, Zoe, and other Agoric tools and
+software, you need to install the following. 
+
+<table border="1">
+  <tbody>
+  <th><b>Step</b></th>
+  <th><b>Action</b></th>
+  <th><b>Explanation</b></th>
+  <tr>
+    <td><center>1</center></td>
+    <td><a href="https://nodejs.org/">Install Node.js</a>, version 10.16.0 or higher</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td><center>2</center></td>
+    <td><a href="https://classic.yarnpkg.com/en/docs/install#mac-stable">Install Yarn 1</a></td>
```

Comment:
"install yarn" isn't enough instruction. Tell them how to do it with some particular package manager, or what to download and install. Yeah, doing this in a platform-neutral way isn't easy. Maybe that means we need a different version for Mac, Windows, Linux, or an intro paragraph giving directions so everyone can start in the same place.

AFAIC, pick a random installer, and anyone who has a different favorite is sophisticated enough to figure out the differences.

Reply:
"Install yarn" isn't sufficient, but the link is. It goes to what amounts to yarn install central, where you can select one of around 8 operating systems to get instructions and downloads needed to install for that OS. I'll clarify in the explanation column about the link.